spanner/api/Spanner.Samples/QueryJsonbDataUsingParameterAsyncPostgre.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_postgresql_jsonb_query_parameter]
+
+using Google.Cloud.Spanner.Data;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+public class QueryJsonbDataUsingParameterAsyncPostgreSample
+{
+    public async Task<List<VenueInformation>> QueryJsonbDataUsingParameterAsyncPostgre(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+        using var connection = new SpannerConnection(connectionString);
+        // Get all the venues with a rating greater than 2.
+        using var command = connection.CreateSelectCommand(
+            "SELECT venueid, details FROM VenueInformation WHERE CAST(details ->> 'rating' AS INTEGER) > $1",
```

Comment:
Is it clear from the context of this sample that details is a JSON? If not, a note here showing the structure of details could be useful.

Reply:
Sure. I will add the following: Details is a column of type Jsonb. The data persisted in this column has the following structure. 
Details = [{
                    "name": "string",
                    "available": true,
                    "rating": int  // This field is optional. 
                }]
Please let me know if any changes are needed. Thanks.

Reply:
Yep, that seems right, only thing is, I believe that Details does not necessarily have the same structure for all rows, right? So instead of saying "The data persisted in this column has the following structure" I would say "Some of the data persisted in this column ...."